src/mesido/workflows/grow_workflow.py

```diff
@@ -309,6 +320,9 @@ def priority_completed(self, priority):
                 self.solver_stats,
             )
         )
+        if priority == 1 and self.objective_value>1e-12:
```

Comment:
Can we change the limit value here, I think 1.e-6 is the default tolerance of the solver

Reply:
I agree, changed.

Reply:
thmx

Reply:
Would you agree to also abort the second stage if heatmatching in the first stage failed?